client/flutter/lib/pages/onboarding/location_sharing_page.dart

```diff
@@ -1,81 +0,0 @@
-import 'package:who_app/api/who_service.dart';
-import 'package:who_app/generated/l10n.dart';
-import 'package:who_app/pages/onboarding/permission_request_page.dart';
-import 'package:flutter/cupertino.dart';
-import 'package:location/location.dart';
-import 'package:s2geometry/s2geometry.dart';
```

Comment:
Remove the location permission and s2geometry deps from pubspec.yaml plz

Reply:
this is coming in a separate PR 😄

Reply:
Done